test/e2e/storage/testsuites/volume_expand.go

```diff
@@ -143,7 +143,7 @@ func (v *volumeExpandTestSuite) defineTests(driver TestDriver, pattern testpatte
 			newSize := currentPvcSize.DeepCopy()
 			newSize.Add(resource.MustParse("1Gi"))
 			e2elog.Logf("currentPvcSize %v, newSize %v", currentPvcSize, newSize)
-			l.resource.pvc, err = ExpandPVCSize(l.resource.pvc, newSize, f.ClientSet)
+			_, err = ExpandPVCSize(l.resource.pvc, newSize, f.ClientSet)
 			framework.ExpectError(err, "While updating non-expandable PVC")
```

Comment:
I know this is kinda long winded but I think we should rather do this:

```
newPVC, err := ExpandPVCSize(l.resource.pvc, newSize, f.ClientSet)
framework.ExpectNoError(err, "While updating pvc for more size")
l.resource.pvc = newPVC
```

And do this wherever `ExpandPVCSize` is called. My reasoning is - this can "potentially" happen in other places where we are assigning result of `ExpandPVCSize` to `l.resource.pvc` too and we should update those call sites as well, just to be safe.

Reply:
it is probably fine to just do `_, err = ExpandPVCSize()` in this place but I think other call sites of `ExpandPVCSize` is problematic too and worth fixing.

Reply:
I agree, will fix all places

Reply:
done, PTAL @gnufied